<launch>
    <node pkg="move_base" type="move_base" name="move_base" output="screen">
        <param name="base_global_planner" value="navfn/NavfnROS"/>
        <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>
        <rosparam file="$(find ashwini)/config/costmap_common_params.yaml" command="load"/>
        <rosparam file="$(find ashwini)/config/global_costmap_params.yaml" command="load"/>
        <rosparam file="$(find ashwini)/config/local_costmap_params.yaml" command="load"/>
        <rosparam file="$(find ashwini)/config/dwa_local_planner_params.yaml" command="load"/>
    </node>
</launch>

